disasterinfosite/templates/index.html

```diff
@@ -48,12 +48,14 @@ <h2 class="project-title caps">
       <h3 class="project-subtitle">{{ settings.site_description }}</h3>
     </div>
     {% block info-instructions %}
+      <div class="information-container">
       <div class="information-instructions">
```

Comment:
This caused a conflict with the role additions you made earlier - clearing it up now.